doc/releases/changelog-0.20.0.md

```diff
@@ -399,50 +431,31 @@
-  [(#1807)](https://github.com/PennyLaneAI/pennylane/pull/1807)
-  [(#1969)](https://github.com/PennyLaneAI/pennylane/pull/1969)
-
-  - Internally, if multiple circuits are generated for simultaneous execution, they
-    will be packaged into a single job for execution on the device. This can lead to
-    significant performance improvement when executing the QNode on remote
-    quantum hardware or simulator devices with parallelization capabilities.
-
-  - Custom gradient transforms can be specified as the differentiation method:
-
-    ```python
-    @qml.gradients.gradient_transform
-    def my_gradient_transform(tape):
-        ...
-        return tapes, processing_fn
+* `qml.math.scatter_element_add` now supports adding multiple values at
+  multiple indices with a single function call, in all interfaces
+  [(#1864)](https://github.com/PennyLaneAI/pennylane/pull/1864)
 
-    @qml.qnode(dev, diff_method=my_gradient_transform)
-    def circuit():
-    ```
+  For example, we may set five values of a three-dimensional tensor
+  in the following way:
 
-  - Arbitrary :math:`n`-th order derivatives are supported on hardware using gradient transforms
-    such as the parameter-shift rule. To specify that an :math:`n`-th order derivative of a QNode
-    will be computed, the `max_diff` argument should be set. By default, this is set to 1
-    (first-order derivatives only). Increasing this value allows for higher order derivatives to be
-    extracted, at the cost of additional (classical) computational overhead during the backwards
-    pass.
+  ```pycon
+  >>> X = tf.zeros((3, 2, 9), dtype=tf.float64)
+  >>> indices = [(0, 0, 1, 2, 2), (0, 0, 0, 0, 1), (1, 3, 8, 6, 7)]
+  >>> values = [0.1 * i for i in range(5)]
+  >>> qml.math.scatter_element_add(X, indices, values)
+  <tf.Tensor: shape=(3, 2, 9), dtype=float64, numpy=
+  array([[[0., 1., 0., 2., 0., 0., 0., 0., 0.],
+          [0., 0., 0., 0., 0., 0., 0., 0., 0.]],
 
-  - When decomposing the circuit, the default decomposition strategy `expansion_strategy="gradient"`
-    will prioritize decompositions that result in the smallest number of parametrized operations
-    required to satisfy the differentiation method. While this may lead to a slight increase in
-    classical processing, it significantly reduces the number of circuit evaluations needed to
-    compute gradients of complicated unitaries.
+         [[0., 0., 0., 0., 0., 0., 0., 0., 3.],
+          [0., 0., 0., 0., 0., 0., 0., 0., 0.]],
 
-    To return to the old behaviour, `expansion_strategy="device"` can be specified.
+         [[0., 0., 0., 0., 0., 0., 4., 0., 0.],
+          [0., 0., 0., 0., 0., 0., 0., 5., 0.]]])>
+  ```
 
-  Note that the old QNode remains accessible at `@qml.qnode_old.qnode`, however this will
-  be removed in the next release.
+* All instances of `str.format` have been replace with f-strings.
+  [(#1970)](https://github.com/PennyLaneAI/pennylane/pull/1970)
```

Comment:
I can't comment on it, but below on line 499, there is a rendering issue due to an incorrect indentation:

![image](https://user-images.githubusercontent.com/2959003/145780603-40229258-a8dc-49e7-b8dc-3b709bc06384.png)